backend/main/src/Shared/Adapters/HonoAdapter.ts

```diff
@@ -49,18 +49,35 @@ export const honoOkResponseAdapter = (
 
 /**
  * This is the just wrapper of the Hono.
- * This returns 500 Bad Request.
- * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+ * This returns 201 OK.
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
  * @param c
- * @param errorMessage - the error message to return.
+ * @param data
  */
-export const honoInternalServerErrorAdapter = (
+export const honoCreatedResponseAdapter = (
 	c: Context,
-	errorMessage: string,
+	data: object | string | null,
 ): Response => {
+	if (typeof data === "string") {
+		c.header("Content-Type", "text/json");
+		c.status(201);
+		return c.text(data);
+	}
+	c.header("Content-Type", "application/json");
+	c.status(201);
+	return c.json(JSON.stringify(data));
+};
+
+/**
+ * This is the just wrapper of the Hono.
+ * This returns 404 Not Found.
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
```

Comment:
What this comment says is different from the implementation.
```suggestion
 * This returns 405 Not Allowed.
 * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405
 ```